### PR TITLE
[chromaprint] add new port with version 1.5.0

### PR DESCRIPTION
**Pull request by Be-ing**
**Describe the pull request**

- What does your PR fix?
Adding port for [chromaprint](https://github.com/acoustid/chromaprint)

- Which triplets are supported/not supported? Have you updated the CI baseline?
all

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
yes

### CODE REVIEW

**Comment by Be-ing** [comment]
x64_osx build has been stalled for hours. Is there an issue with your infrastructure?

File: ports/chromaprint/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,15 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO acoustid/chromaprint
+  REF v1.5.0
+  SHA512 333114949928abdf5d4b11aba1db6ec487eebe526324c68d903b3fa80a3af87a28d942af765a2f873e63a1bf222b658b6438cd10cde4446f61b26ea91f537469
+)
```
Comment (NancyLi1013) [inline ports/chromaprint/portfile.cmake:6]:
```suggestion
vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO acoustid/chromaprint
    REF v1.5.0
    SHA512 333114949928abdf5d4b11aba1db6ec487eebe526324c68d903b3fa80a3af87a28d942af765a2f873e63a1bf222b658b6438cd10cde4446f61b26ea91f537469
)
```

File: ports/chromaprint/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
```
Comment (NancyLi1013) [inline ports/chromaprint/portfile.cmake:9]: Why not use `PREFER_NINJA `here? Is there some problem if using `PREFER_NINJA `?

File: ports/chromaprint/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+)
+vcpkg_install_cmake()
+vcpkg_fixup_pkgconfig()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+file(INSTALL ${SOURCE_PATH}/LICENSE.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/chromaprint RENAME copyright)
```
Comment (NancyLi1013) [inline ports/chromaprint/portfile.cmake:15]:
```suggestion
file(INSTALL ${SOURCE_PATH}/LICENSE.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/chromaprint/portfile.cmake (hunk 2 above)
Comment (Be-ing) [reply ports/chromaprint/portfile.cmake:9]: I'll add `PREFER_NINJA`. Why isn't that the default? I don't understand why every portfile.cmake has to add that manually.

**Comment by Be-ing** [comment]
x64_osx CI has not been run in 20 hours. Is it down?

**Comment by Be-ing** [comment]
CI passed now :heavy_check_mark:

File: ports/chromaprint/portfile.cmake (hunk 2 above)
Comment (ras0219-msft) [reply ports/chromaprint/portfile.cmake:9]: Backwards compatibility; originally we used CMake's default of MSBuild on windows.

**Comment by ras0219-msft** [comment]
LGTM, thanks for the new port!

File: ports/chromaprint/portfile.cmake (hunk 2 above)
Comment (Be-ing) [reply ports/chromaprint/portfile.cmake:9]: Thanks for clarifying.

**Pull request merged**